ts/src/bitmex.ts

```diff
@@ -1536,7 +1536,7 @@ export default class bitmex extends Exchange {
         }
         const until = this.safeInteger2 (params, 'until', 'endTime');
         if (until !== undefined) {
-            params = this.omit (params, [ 'until' ]);
+            params = this.omit (params, [ 'until', 'endTime' ]);
```

Comment:
@cheunhong So usually we don't omit the exchange-specific key, the only fix we can apply here is to read only `until` from `params` and not `endTime` since it is not the unified syntax